sneaker-frontend/src/MainScreen.js

```diff
@@ -116,9 +110,9 @@ function MainScreen() {
             });
     }
 
-    
-    
-    const DraggableShoeCard = ({ user, shoe, index, moveShoe, greyOutNonFav}) => {
+
+
+    const DraggableShoeCard = ({ user, shoe, index, moveShoe, greyOutNonFav }) => {
```

Comment:
user not used inside the function right? we can remove it

Reply:
Correct, is removed